lib/matplotlib/ticker.py

```diff
@@ -2079,13 +2235,13 @@ def decade_up(x, base=10):
     return base ** lx
 
 
-def is_decade(x, base=10):
+def is_decade(x, base=10, *, tol=None):
```

Comment:
perhaps name this rtol? (well it's not *strictly* a relative tolerance, but it's closer to a relative tolerance than to an absolute one)

Reply:
It is a absolute tolerance in logspace, then, for me, it is a absolute tolerance.

Fixed.